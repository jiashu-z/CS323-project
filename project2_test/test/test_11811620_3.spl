struct A
{
  int num1;
  int arr1[1];
};

struct B
{
};

int test_11811620_3()
{
  int a;
  float b;
  char c;
  struct A d;
  int e[1][1];
  struct A f[1][1][1];
  struct B g;

  a = b;
  c = a;
  e = f;
  e = f[0];
  f[0][0][0] = g;
  f[0][0][0] = d;
  d = f[0][0][0];
  f[0][0][0].num1 = a;
  f[0][0][0].arr1 = e;
  f[0][0][0].arr1 = e[0];

  1 = a;

  d = 1 + d;
  e = 1 + e;
  return 0;
}