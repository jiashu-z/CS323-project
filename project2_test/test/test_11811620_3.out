Error type 5 at Line 21: unmatching type on both sides of assignment
Error type 5 at Line 22: unmatching type on both sides of assignment
Error type 5 at Line 23: unmatching type on both sides of assignment
Error type 5 at Line 24: unmatching type on both sides of assignment
Error type 5 at Line 25: unmatching type on both sides of assignment
Error type 5 at Line 29: unmatching type on both sides of assignment
Error type 6 at Line 32: left side in assignment is rvalue
Error type 7 at Line 34: binary operation on non-number variables
Error type 5 at Line 34: unmatching type on both sides of assignment
Error type 7 at Line 35: binary operation on non-number variables
Error type 5 at Line 35: unmatching type on both sides of assignment